src/api-mock/vue-mock.js

```diff
@@ -0,0 +1,73 @@
+import Vue from 'vue'
+import vueResource from 'vue-resource'
+import apiRoutes from 'mock/api-routes'
+import Url from 'url'
+
+let routes = apiRoutes
+
+Vue.use(vueResource)
+
+if (process.env.NODE_ENV !== 'production' && window.concatMockRoutes(routes)) {
```

Comment:
```suggestion
if (process.env.NODE_ENV != 'production' && window.concatMockRoutes(routes)) {
```

Reply:
maksudny gan ? untuk js !== is no problem

Reply:
!== for strict comparison. != for lose comparison